src/pulik/atm/Main.java

```diff
@@ -5,31 +5,27 @@
 import pulik.atm.cards.Card;
 import pulik.atm.cards.DebitCard;
 import pulik.atm.machines.TellerMachine;
+import pulik.atm.machines.TerminalATM;
 
 public class Main {
-	public static void main(String[] args) {
+	public static void main(String[] args) throws Exception {
```

Comment:
I'm not even sure this would be able to run with the `throws` in the `main` method's declaration? The `main()` method shouldn't throw any exceptions. If any checked exception can occur, it should be caught in `try-catch` block at least in the `main()` method.